packages/devextreme/js/events/pointer/base.js

```diff
@@ -35,11 +36,8 @@ const BaseStrategy = Class.inherit({
             timeStamp: browser.mozilla ? (new Date()).getTime() : e.timeStamp
         };
 
-        const originalEvent = e.originalEvent;
-        if(originalEvent?.target?.shadowRoot) {
-            const path = originalEvent.path ?? originalEvent.composedPath?.();
-            event.target = path[0];
-        }
+        const target = getEventTarget(e);
+        event.target = target;
```

Comment:
why do we need a separate var for target?
couldn't we just write 
event.target = getEventTarget(e);

Reply:
Yeah, we can, left it this way just for debugging simplifying